src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -128,17 +128,20 @@ public void ValidateConfig()
                         HttpStatusCode.ServiceUnavailable,
                         DataApiBuilderException.SubStatusCodes.ErrorInInitialization);
 
-                if (string.IsNullOrEmpty(cosmosDbNoSql.Schema))
+                if (string.IsNullOrEmpty(cosmosDbNoSql.GraphQLSchema))
```

Comment:
Good find! in order to ensure there's no regression in future, can we add a test that does this?

`Modified the test config file to remove the schema, verified that the late config tests were failing`

Reply:
Sure, but it would required either a second configuration file for cosmos which will then have to be maintained, or to parse the config file in the test to change that variable. Any preference on the approach?

Reply:
The latter is preferable - parse it, and change that variable - then use it to configure

Reply:
similar to how in ConfigurationTests.cs `ConfigurationPostParameters configParams = GetCosmosConfigurationParameters();` sets the values for the CosmosOptions

Reply:
also worth a code comment here calling out that `GraphQLSchema` (and not `Schema`) will be populated when configuration is received post-startup